packages/ndk-multilib/build.sh

```diff
@@ -37,11 +37,60 @@ prepare_libs() {
 	fi
 
 	cp $LIBATOMIC/libatomic.a $TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/$SUFFIX/lib/
+
+	cp $LIBATOMIC/libunwind.a $TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/$SUFFIX/lib/
+}
+
+add_cross_compiler_rt() {
+	# FIXME: Should we get libcompiler-rt from libllvm package?
```

Comment:
No need, the one from the NDK is fine.

Reply:
OK. I will remove this line from the `build.sh`.